CITATION.cff

```diff
@@ -0,0 +1,84 @@
+cff-version: 1.2.0
+title: A Python Library for NeXus-Compliant Experimental Data Conversion and Integration with NOMAD Platform
+message:
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+version: 0.7.2
+authors:
+  - given-names: Sherjeel
+    family-names: Shabih
+    orcid: 'https://orcid.org/0009-0008-6635-4465'
+  - given-names: Florian
+    family-names: Dobener
+    orcid: 'https://orcid.org/0000-0003-1987-6224'
+  - given-names: Markus
+    family-names: Kühbach
```

Comment:
Why is Markus K now third author? I think it should be Sherjeel and Florian 1 and 2. And then there's two options: Markus S as 3 or everybody after 2 by alphabetical order

Reply:
Markus Scheidgen is completely missing, he must be added

Reply:
On the last Friday, I discussed it in our discord channel. No one has any complaints there, right?

Reply:
And he is also the main idea provider and writer of the `NeXus parser`, which is one of the main tool in pynxtools.

Reply:
Markus Scheidgen is on the list of authors.

Reply:
But you have Markus Kühbach as third and not Markus Scheidgen. List should be
1. Sherjeel
2. Florian
3. Markus Scheidgen
4.-N. everybody else in alphabetical order

Reply:
Ah, Sorry, you meant MarkusS.

Reply:
Then after the first two contributors, I should put all others in alphabetic order. Right?

Reply:
yes